assets/css/style.css

```diff
@@ -0,0 +1,83 @@
+@media only screen and (max-width: 768px) {
+  body {
+    overflow: hidden;
+  }
+
+  .navbar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .nav-branding {
+    color: #28352f;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 6vw;
+    line-height: 2.8rem;
+    text-decoration: none;
+    margin-left: 2.4vh;
+    margin-top: 5.4vh;
+  }
+
+  .humburger {
+    display: block;
+    margin-right: 1vh;
+    margin-top: 5.4vh;
+  }
+
+  .humburger .bar {
+    display: block;
+    width: 25px;
+    height: 3px;
+    color: brown;
+    margin: 5px auto;
+    -webkit-transition: all 0.3s ease-in-out;
+    transition: all 0.3s ease-in-out;
+    background-color: #28352f;
+  }
+
+  .headline {
+    position: relative;
+    background-image: url(/assets/images/header-bg.png);
+    background-repeat: no-repeat;
+    background-size: 90%;
```

Comment:
- [ ] Great Job Implementing the background image from Figma, the background-size is not correct, I would suggest using `background-size: cover; ` so the background cover the whole section



![bg](https://user-images.githubusercontent.com/89282221/184872842-d0465b5a-35fb-42b8-b04f-d51bc3661f22.jpg)

Reply:
- Update background size to : cover